pizza-facts-&-trivia.md

```diff
@@ -24,6 +24,7 @@ We've categorized the facts into different sections to help explore the diverse
 - As of 2017, Domino’s Pizza is the most popular pizza chain in the world. The company has more than 17,000 pizza stores in 90 countries, and this number continues to grow.
 - Anchovies are America's least favorite topping.
 - The longest pizza measures 1,930.39 m (6,333 ft 3.60 in) and was achieved in Fontana, California, USA on 10 June 2017. This record-breaking pizza was made using approximately 17,700 lbs of dough, 5,000 lbs of tomato sauce, and 3,900 lbs of mozzarella cheese.
+- On January 18th, 2023, Pizza Hut, in collaboration with YouTuber Eric Decker (Airrack), claimed the Guinness World Record for the World's Largest Pizza, which measured an astonishing 13,990 square feet. This remarkable achievement marked a significant pizza history milestone.
```

Comment:
This fact is not duplicate and it is `ready to merge`.
**BUT**
You are not mention the issue in your description, so that it shows `Merging is blocked`. 
You PR is Relates to https://github.com/open-sauced/pizza-verse/issues/36.
Add `Relates to #36` inside **Related Tickets & Documents**.

Reply:
Okay. Thank you. I have updated the Related Tickets & Documents section.

Reply:
Hi @Ken-Musau, 
According to the [official Guiness World Record website for largest pizza](https://www.guinnessworldrecords.com/world-records/largest-pizza), this record was achieved on January 19th, 2023 and the pizza measures 13,957.77 ft².

Can you please fix the details so we can merge your PR?
Thank you. :)